From 45c0fa8f2b6b78eb3f2404cb5269a89e46948603 Mon Sep 17 00:00:00 2001
From: Thierry Escande <thierry.escande@vates.tech>
Date: Fri, 5 Apr 2024 10:31:22 +0200
Subject: [PATCH 2/2] Makefile: Add EXTRA_CFLAGS to CFLAGS
Content-Type: text/plain; charset = "utf-8"
Content-Transfert-Encoding: 8bit

This patch allows passing of extra compilation flags from command line
using 'make EXTRA_CFLAGS=-DFOO'.

Signed-off-by: Thierry Escande <thierry.escande@vates.tech>
---
 Makefile | 3 +++
 1 file changed, 3 insertions(+)

diff --git a/Makefile b/Makefile
index d345c11..7c43b11 100644
--- a/Makefile
+++ b/Makefile
@@ -19,6 +19,9 @@ CFLAGS  = -I$(shell pwd)/include
 # _GNU_SOURCE for asprintf.
 CFLAGS += -D_LARGEFILE_SOURCE -D_LARGEFILE64_SOURCE -D_GNU_SOURCE
 
+# EXTRA_CFLAGS can be set through make command line
+CFLAGS += $(EXTRA_CFLAGS)
+
 CFLAGS += $$(pkg-config --cflags libxml-2.0)
 
 CFLAGS += -g -O2 -std=gnu99 \
-- 
2.43.0

